src/components/CurrencyLogo/index.tsx

```diff
@@ -14,6 +14,8 @@ import { getProxyTokenLogo } from 'utils/tokenInfo'
 const StyledNativeCurrencyLogo = styled.img<{ size: string }>`
   width: ${({ size }) => size};
   height: ${({ size }) => size};
+  min-width: ${({ size }) => size};
+  min-height: ${({ size }) => size};
```

Comment:
🤔 để vậy nó có kéo dãn hình nhìn sida ko?

Reply:
kéo là kéo sao fen. chưa hiểu.
này thêm vô để fix 1 số case width của thằng cha bé quá nó mất token logo

Reply:
isn't width enogh?

Reply:
If you have tested this carefully, it's ok to resolve this comment

Reply:
just tested again, it's ok to remove min-height